resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,640 @@
+| GCP region. Can be any region, but should be geographically close to on-premises gateway. | `[REGION2]` | `us-west1` |
+| IP address range for the GCP VPC subnet (`vpn-subnet-1`) | `[RANGE_1]` | `10.0.1.0/24` |
+| IP address range for the GCP VPC subnet (`vpn-subnet-2`) | `[RANGE_2]` | `10.0.2.0/24` |
+| IP address range for the on-premises subnet. You will use this range when creating rules for inbound traffic to GCP. | `[IP_ON_PREM_SUBNET]` | `192.168.1.0/24` |
+| External static IP address for the first internet interface of \<vendor-name>\<product-name>  | `[ON_PREM_GW_IP_0]` | `199.203.248.181` |
+| External static IP address for the second internet interface of \<vendor-name>\<product-name> | `[ON_PREM_GW_IP_1]` | `199.203.248.182` |
+| HA VPN gateway                          | `[GW_NAME]`                 | `ha-vpn-gw-a`                       |
+| Cloud Router name (for dynamic routing) | `[ROUTER_NAME]`             | `router-a`                          |
+| Google ASN                              | `[GOOGLE_ASN]`              | `65001`                             |
+| Peer ASN                                | `[PEER_ASN]`                | `65002`                             |
+| External VPN gateway resource           | `[PEER_GW_NAME]`            | `peer-gw`                           |
+| First VPN tunnel                        | `[TUNNEL_NAME_IF0]`         | `tunnel-a-to-on-prem-if-0`          |
+| Second VPN tunnel                       | `[TUNNEL_NAME_IF1]`         | `tunnel-a-to-on-prem-if-1`          |
+| First BGP peer interface                | `[ROUTER_INTERFACE_NAME_0]` | `bgp-peer-tunnel-a-to-on-prem-if-0` |
+| Second BGP peer interface               | `[ROUTER_INTERFACE_NAME_1]` | `bgp-peer-tunnel-a-to-on-prem-if-1` |
+| BGP interface netmask length            | `[MASK_LENGTH]`             | `/30`                               |
+
+## Configure the GCP side
+
+This section covers how to configure HA VPN.
+
+There are two ways to create HA VPN gateways on GCP: using the GCP Console and using
+[`gcloud` commands](https://cloud.google.com/sdk/).
+
+This section describes how to perform the tasks using `gcloud` commands. 
+
+### Initial tasks
+
+Complete the following procedures before configuring a GCP HA VPN gateway and tunnel.
+
+These instructions create a [custom mode](https://cloud.google.com/vpc/docs/vpc#subnet-ranges) 
+VPC network with one subnet in one region and another subnet in another region.
+
+#### Create a custom VPC network
+
+If you haven't already, create a VPC network with this command:
+
+    gcloud compute networks create [NETWORK] \
+    --subnet-mode [SUBNET_MODE]  \
+    --bgp-routing-mode [BGP_ROUTING_MODE]
+
+Replace the placeholders as follows:
+
+- `[NETWORK]`: Assign a network name.
+- `[SUBNET_MODE]`: Set as `custom`.
+- `[BGP_ROUTING_MODE]`: Set as `global`.
+
+The command should look similar to the following example:
```

Comment:
This is still incorrect. You can add the command output from the real guide as an example, or remove the output.

Reply:
I'll leave it to Ashish to decide what to do with these sections, but I am confused by your response, Julie.

The section begins by giving the command with some placeholders, then explains the placeholders, and then shows how the command would look with some example values instead of the palceholders. It's a little overly hand-holding for my tastes, but I don't understand why you are referring to it as incorrect or why you are referring to this as output.

Reply:
Since this is a guide idea here is to give the actual command so users can just copy paste it on their terminal, not the output of the command as shown in official docs. I agree with your comment Todd, it is little too much for me as well but I am just trying to keep it similar to Classic VPN template.

Reply:
I see, Todd. Ashish, I'll leave it up to whether or not you want to leave these examples in the guide.

Reply:
Thanks both for the review.